modules/ovis/include/opencv2/ovis.hpp

```diff
@@ -236,6 +245,20 @@ class CV_EXPORTS_W WindowScene {
      */
     CV_WRAP virtual void setCameraLookAt(const String& target, InputArray offset = noArray()) = 0;
 
+	/**
+     * convenience method to orient an entity to a point in world coordinates
+	 * @param origin entity name
+     * @param target targetpoint vector
+     */
+	CV_WRAP virtual void setEntityLookAt(const String& origin, InputArray target) = 0;
```

Comment:
combine both functions into one as in `setCameraLookAt` to allow an lookAt offset. Conversely you can implement lookat for an arbitrary destination if `target == ""`.